packages/web-client/app/components/card-pay/merchant-card/index.hbs

```diff
@@ -0,0 +1,29 @@
+<article
+  class="merchant-card"
+  style={{css-var merchant-logo=@logo}}
+  data-test-merchant-card-logo={{@logo}}
+  data-test-merchant-card={{@id}}
+>
+  <header class="merchant-card__header">
+    <div class="merchant-card__header-network">on {{@network}}</div>
+    <div class="merchant-card__header-address">{{truncate-middle @address}}</div>
+  </header>
+  <section class="merchant-card__content">
+    <div class="merchant-card__merchant">
+      <div class="merchant-card__logo">{{truncate @name 1 false}}</div>
```

Comment:
I know it is small, but we should keep an eye out to see if this should be its own component.

Reply:
Added a note for this